packages/shared/comms/index.ts

```diff
@@ -890,11 +891,33 @@ function subscribeToRealmChange(store: Store<RootState>) {
   )
 }
 
+async function getPeerParameters(): Promise<PeerParameters> {
+  const commsServer = getCommsServer(store.getState())
```

Comment:
Can we move the endpoint to a lambda? To decouple the configuration of comms from the actual implementation

Reply:
I see. Still, I think we should add those endpoints as part of solving the particular problems they will solve. Mixing problems can lead us to try and solve something that we don't yet have defined.

Thanks for approving!